myuw/templates/handlebars/card/summary/section_panel.html

```diff
@@ -1,17 +1,21 @@
 {% load templatetag_handlebars %}
     {% tplhandlebars "summary_section_panel" %}
     <div class="row">
+        {{#unless under_disclosure}}
         <div class="col-sm-3">
             <h4 class="myuw-card-row-heading" style="display:inline-block;
                 margin-bottom:0px;"><div class="c{{color_id}} simplesquare" aria-hidden="true"></div><a href="/teaching/{{ @root.year }},{{ @root.quarter }}#{{ curriculum_abbr }}-{{ course_number }}-{{ section_id }}"
-                class="future-nav-link" future-nav-target="{{@root.year}},{{@root.quarter}},{{curriculum_abbr}}-{{course_number}}-{{section_id}}">{{#unless under_disclosure}}{{curriculum_abbr}} {{/unless}}{{course_number}} {{section_id}}</a></h4>
-            {{#unless under_disclosure}}<span class="meeting-type">{{ meetings.0.type }}</span>{{/unless}}
+                class="future-nav-link" future-nav-target="{{@root.year}},{{@root.quarter}},{{curriculum_abbr}}-{{course_number}}-{{section_id}}">{{curriculum_abbr}} {{course_number}} {{section_id}}</a></h4>
+            <span class="meeting-type">{{ meetings.0.type }}</span>
         </div>
 
         <div class="col-sm-9">
+        {{/unless}}
```

Comment:
should this end of {{/unless}} be on line 11?

Reply:
Yes, essentially I'm putting the top level classes into our grid system, while trying to take classes that are in the disclosures out of a grid system and only into a table.